NOTICE.txt

```diff
@@ -436,6 +436,21 @@ Copyright 2018 CoreOS, Inc
 This product includes software developed at CoreOS, Inc.
 (http://www.coreos.com/).
 
+--------------------------------------------------------------------
+Dependency: github.com/coreos/go-systemd
```

Comment:
This library is already vendored in the root-level `./vendor` (see some lines above here), we should reuse it.